src/chi/components/modal/modal.scss

```diff
@@ -15,29 +17,92 @@
   overflow: hidden;
   pointer-events: all;
   position: relative;
+  top: 0;
   width: 100%;
 
+  &.-active {
+    display: block;
+  }
+
+  &.-animated {
+    display: block;
+    &:not(.-active) {
+      top: -1000px;
+    }
+
+    transition-duration: $animation-duration;
+    transition-property: top;
+    transition-timing-function: ease-in-out;
+
+    &.-transitioning {
+      display: block;
+    }
+  }
+
   .-close {
     position: absolute;
     right: 0.75rem;
     top: 0.8125rem;
   }
 
+  .-closed {
+    display: none;
+  }
+
   &__header {
     box-shadow: inset 0 -1px 0 0 rgba(set-color(black), 0.12);
     min-height: 3.5rem;
     padding: 1rem;
     position: relative;
+    top: 0;
+    width: 100%;
 
-    &.-noBorder {
-      box-shadow: none;
+    &.-active {
+      display: block;
     }
 
-    .a-modal__title {
-      font-size: $text-large;
-      font-weight: bold;
-      line-height: $line-height-large;
-      margin: 0;
+    &.-animated {
+      display: block;
+      &:not(.-active) {
+        top: -1000px;
+      }
+
+      transition-duration: $animation-duration;
+      transition-property: top;
+      transition-timing-function: ease-in-out;
+
+      &.-transitioning {
+        display: block;
+      }
+    }
+
+    .-close {
+      position: absolute;
+      right: 0.75rem;
+      top: 0.8125rem;
+    }
+
+    .-closed {
+      display: none;
+    }
+
+    &__header {
+      box-shadow: inset 0 -1px 0 0 rgba(set-color(black), 0.12);
+      min-height: 3.5rem;
+      padding: 1rem;
+      position: relative;
+
+      &.-noBorder {
+        box-shadow: none;
+      }
+
+      .a-modal__title {
```

Comment:
@davidojedacl doesn't look like this is getting applied

Reply:
i will check